src/v2/Apps/Auction/__tests__/AuctionApp.jest.tsx

```diff
@@ -309,13 +325,21 @@ describe("AuctionApp", () => {
       )
     })
 
-    it("shows auctions rail", () => {
+    it("renders the auction rail", () => {
       const wrapper = getWrapper({
         Sale: () => ({
           status: "preview",
+          eligibleSaleArtworksCount: 0,
         }),
       })
       expect(wrapper.find("AuctionCurrentAuctionsRail").length).toBe(1)
     })
+
+    it("does not render the auction rail", () => {
+      const wrapper = getWrapper({
+        Sale: () => ({}),
+      })
+      expect(wrapper.find("AuctionCurrentAuctionsRail").length).toBe(0)
+    })
```

Comment:
I added this to make the CI pass, but the conditions seem backward to me? I was a little confused so I wanted to push to commit and ask.

Reply:
Thank you!